pkg/config/config.go

```diff
@@ -46,20 +36,9 @@ func New() Config {
 		PostgresContainerName:   getEnv("POSTGRES_CONTAINER_NAME", "postgresql"),
 		OauthProxyContainerName: getEnv("OAUTH_PROXY_CONTAINER_NAME", "ups-oauth-proxy"),
 
-		UPSImageStreamName: getEnv("UPS_IMAGE_STREAM_NAME", "ups-imagestream"),
-		UPSImageStreamTag:  getEnv("UPS_IMAGE_STREAM_TAG", "2.3"),
-
-		OauthProxyImageStreamName: getEnv("OAUTH_PROXY_IMAGE_STREAM_NAME", "ups-oauth-proxy-imagestream"),
-		OauthProxyImageStreamTag:  getEnv("OAUTH_PROXY_IMAGE_STREAM_TAG", "latest"),
-
-		PostgresImageStreamNamespace: getEnv("POSTGRES_IMAGE_STREAM_NAMESPACE", "openshift"),
-		PostgresImageStreamName:      getEnv("POSTGRES_IMAGE_STREAM_NAME", "postgresql"),
-		// Used both to set the tag, and also for the "POSTGRES_VERSION" in the Secret
-		PostgresImageStreamTag: getEnv("POSTGRES_IMAGE_STREAM_TAG", "10"),
-
-		// these are used when the image stream does not exist and created for the first time by the operator
-		UPSImageStreamInitialImage:        getEnv("UPS_IMAGE_STREAM_INITIAL_IMAGE", "quay.io/aerogear/unifiedpush-configurable-container:2.3"),
-		OauthProxyImageStreamInitialImage: getEnv("OAUTH_PROXY_IMAGE_STREAM_INITIAL_IMAGE", "quay.io/openshift/origin-oauth-proxy:4.2.0"),
+		UPSImage:        getEnv("UPS_IMAGE", "quay.io/aerogear/unifiedpush-configurable-container:2.3"),
```

Comment:
Should we have UPS_IMAGE? Based on this `3. A version of the operator should handle one version of the operand (the service version).` my understanding is that we don't want to make this configurable. Updating the operator version should be the path to upgrade the service right?

Reply:
Got rid of them now